timesketch/lib/utils.py

```diff
@@ -332,6 +334,41 @@ def read_and_validate_csv(
                 _scrub_special_tags(row)
                 # Remove all NAN values from the pandas.Series.
                 row.dropna(inplace=True)
+                MAX_TIMESTAMP_DIFFERENCE = 100000  # 100 seconds
```

Comment:
As discussed offline - Instead of rewriting the fualty datetime we should error out and let the user fiox the broken input.